beaker_kernel/lib/context.py

```diff
@@ -21,8 +25,9 @@
 
 
 class BaseContext:
-    beaker_kernel: "LLMKernel"
-    subkernel: "BaseSubkernel"
+    # TODO: Delete below
+    #beaker_kernel: "LLMKernel"
+    #subkernel: "BaseSubkernel"
```

Comment:
These lines should still exist here. We're defining that they are class attributes and defining the types and they both still exist and will still exist on the context, unless I'm mistaken.